agent/statemanager/state_manager.go

```diff
@@ -83,7 +83,13 @@ const (
 	//   b) Add 'StartTime' field to 'api.container.Container'
 	//   c) Add 'StopTime' field to 'api.container.Container'
 	// 21) Add 'target' field to the Secret struct
-	// 22) Add 'attachmentType' field to 'api.ENIAttachment'
+	// 22)
+	// 	 a) Add 'attachmentType' field to 'api.ENIAttachment'
+	//	 b) Add 'InterfaceAssociationProtocol' field to 'api.ENI'
+	//	 c) Add 'InterfaceVlanProperties' field to 'api.ENI'
+	//	 d) Add 'NetworkInterfaceVlanProperties' struct
```

Comment:
d) and e) are not needed since they are structs for acs model and won't be saved in state file